ci/azure_pipelines/templates/info.yml

```diff
@@ -0,0 +1,43 @@
+# This template includes some steps to check
+# and gather information about the system
+steps:
+- script: |
+    # the hash was computed in Mac
+    if [[ $AGENT_OS == "Darwin" ]]; then 
+      pushd $BUILD_SOURCESDIRECTORY
+      EXPECTED_HASH_TRAVIS='f98de9357fe7e679c7e0522ba84bbfd7a959b7b0d368427630d3ccd0bffd3097'
+      CURRENT_HASH_TRAVIS=$(shasum -a 256 ./.travis.yml | awk '{print $1}')
+      if [[ $EXPECTED_HASH_TRAVIS != $CURRENT_HASH_TRAVIS ]]; then 
+        echo "The original Travis file of the project has changed"
+        echo "Expected Hash: $EXPECTED_HASH_TRAVIS"
+        echo "Current Hash:  $CURRENT_HASH_TRAVIS"
+        echo "Please check the changes, change the azure pipelines acordingly and update the sha256"
```

Comment:
Nit: accordingly* here and below

Reply:
Done!
I also addressed all the new lines that were missing.
Thx